chart/values.yaml

```diff
@@ -0,0 +1,40 @@
+name: sailoperator
+deploymentName: istio-operator
+service:
+  port: 8443
+serviceAccountName: istio-operator
+
+csv:
+  categories: OpenShift Optional, Integration & Delivery, Networking, Security
+  capabilities: Seamless Upgrades
+  description: Experimental operator for installing Istio service mesh
+  longDescription: |-
+    This is an experimental operator for installing Istio service mesh.
+
+    This version of the operator supports the following Istio versions:
+
+    - v1.20.1
+    - v1.20.0
+    - v1.19.5
+    - latest (e6756e19)
+    - gwAPIControllerMode
```

Comment:
This is generated from the versions.yaml file by [update-version-list.sh](https://github.com/Maistra/istio-operator/blob/b73f1a3146715bf9c37eb43fdce4bf470ae688b5/hack/update-version-list.sh#L44-L60), so you'll have to change that code so it updates the values.yaml file instead.

Reply:
thanks! done